Description: add support for kfreebsd in configure
diff --git a/autoconf/configure.in b/autoconf/configure.in
index fbb3c6b..da017a0 100644
--- a/autoconf/configure.in
+++ b/autoconf/configure.in
@@ -359,6 +359,7 @@ case $host in
    *-*-hpux*)    AC_DEFINE(HAVE_HPUX_OS,,    [Define if building for HPUX])    ;;
    *-*-linux*)   AC_DEFINE(HAVE_LINUX_OS,,   [Define if building for Linux])   ;;
    *-*-freebsd*) AC_DEFINE(HAVE_FREEBSD_OS,, [Define if building for FreeBSD]) ;;
+   *-*-kfreebsd*)AC_DEFINE(HAVE_FREEBSD_OS,, [Define if building for kFreeBSD]) ;;
    *-*-netbsd*)  AC_DEFINE(HAVE_NETBSD_OS,,  [Define if building for NetBSD])  ;;
    *-*-openbsd*) AC_DEFINE(HAVE_OPENBSD_OS,, [Define if building for OpenBSD]) ;;
    *-*-bsdi*)    AC_DEFINE(HAVE_BSDI_OS,,    [Define if building for BSDi])    ;;
diff --git a/configure b/configure
index a6af032..f03eec0 100755
--- a/configure
+++ b/configure
@@ -10483,6 +10483,11 @@ cat >>confdefs.h <<\_ACEOF
 #define HAVE_FREEBSD_OS /**/
 _ACEOF
  ;;
+   *-*-kfreebsd*)
+cat >>confdefs.h <<\_ACEOF
+#define HAVE_FREEBSD_OS /**/
+_ACEOF
+ ;;
    *-*-netbsd*)
 cat >>confdefs.h <<\_ACEOF
 #define HAVE_NETBSD_OS /**/
