Index: apcupsd/autoconf/configure.in
===================================================================
--- apcupsd.orig/autoconf/configure.in
+++ apcupsd/autoconf/configure.in
@@ -350,6 +350,7 @@ case $host in
    *-*-hpux*)    AC_DEFINE(HAVE_HPUX_OS)    ;;
    *-*-linux*)   AC_DEFINE(HAVE_LINUX_OS)   ;;
    *-*-freebsd*) AC_DEFINE(HAVE_FREEBSD_OS) ;;
+   *-*-kfreebsd*)AC_DEFINE(HAVE_FREEBSD_OS) ;;
    *-*-netbsd*)  AC_DEFINE(HAVE_NETBSD_OS)  ;;
    *-*-openbsd*) AC_DEFINE(HAVE_OPENBSD_OS) ;;
    *-*-bsdi*)    AC_DEFINE(HAVE_BSDI_OS)    ;;
Index: apcupsd/configure
===================================================================
--- apcupsd.orig/configure
+++ apcupsd/configure
@@ -10312,6 +10312,10 @@ _ACEOF
 #define HAVE_FREEBSD_OS 1
 _ACEOF
  ;;
+   *-*-kfreebsd*) cat >>confdefs.h <<\_ACEOF
+#define HAVE_FREEBSD_OS 1
+_ACEOF
+ ;;
    *-*-netbsd*)  cat >>confdefs.h <<\_ACEOF
 #define HAVE_NETBSD_OS 1
 _ACEOF
