Description: avoid exclusionary language in configuration files
Index: apcupsd/platforms/etc/changeme.in
===================================================================
--- apcupsd.orig/platforms/etc/changeme.in	2024-06-05 10:29:03.932770903 +0200
+++ apcupsd/platforms/etc/changeme.in	2024-06-05 10:32:02.705765520 +0200
@@ -3,7 +3,7 @@
 # This shell script if placed in /etc/apcupsd
 # will be called by /etc/apcupsd/apccontrol when apcupsd
 # detects that the battery should be replaced.
-# We send an email message to root to notify him.
+# An email notification is sent to the $SYSADMIN email address defined in /etc/apcupsd/apccontrol.
 #
 
 HOSTNAME=`hostname`
Index: apcupsd/platforms/etc/commfailure.in
===================================================================
--- apcupsd.orig/platforms/etc/commfailure.in	2024-06-05 10:29:03.932770903 +0200
+++ apcupsd/platforms/etc/commfailure.in	2024-06-05 10:32:26.489898524 +0200
@@ -3,7 +3,7 @@
 # This shell script if placed in /etc/apcupsd
 # will be called by /etc/apcupsd/apccontrol when apcupsd
 # loses contact with the UPS (i.e. the serial connection is not responding).
-# We send an email message to root to notify him.
+# An email notification is sent to the $SYSADMIN email address defined in /etc/apcupsd/apccontrol.
 #
 
 HOSTNAME=`hostname`
Index: apcupsd/platforms/etc/commok.in
===================================================================
--- apcupsd.orig/platforms/etc/commok.in	2024-06-05 10:29:03.932770903 +0200
+++ apcupsd/platforms/etc/commok.in	2024-06-05 10:32:37.257958797 +0200
@@ -3,7 +3,7 @@
 # This shell script if placed in /etc/apcupsd
 # will be called by /etc/apcupsd/apccontrol when apcupsd
 # restores contact with the UPS (i.e. the serial connection is restored).
-# We send an email message to root to notify him.
+# An email notification is sent to the $SYSADMIN email address defined in /etc/apcupsd/apccontrol.
 #
 
 HOSTNAME=`hostname`
Index: apcupsd/platforms/etc/offbattery.in
===================================================================
--- apcupsd.orig/platforms/etc/offbattery.in	2024-06-05 10:29:03.932770903 +0200
+++ apcupsd/platforms/etc/offbattery.in	2024-06-05 10:32:48.082019406 +0200
@@ -3,7 +3,7 @@
 # This shell script if placed in /etc/apcupsd
 # will be called by /etc/apcupsd/apccontrol when the    
 # UPS goes back on to the mains after a power failure.
-# We send an email message to root to notify him.
+# An email notification is sent to the $SYSADMIN email address defined in /etc/apcupsd/apccontrol.
 #
 
 HOSTNAME=`hostname`
Index: apcupsd/platforms/etc/onbattery.in
===================================================================
--- apcupsd.orig/platforms/etc/onbattery.in	2024-06-05 10:29:03.932770903 +0200
+++ apcupsd/platforms/etc/onbattery.in	2024-06-05 10:33:00.314087944 +0200
@@ -3,7 +3,7 @@
 # This shell script if placed in /etc/apcupsd
 # will be called by /etc/apcupsd/apccontrol when the UPS
 # goes on batteries.
-# We send an email message to root to notify him.
+# An email notification is sent to the $SYSADMIN email address defined in /etc/apcupsd/apccontrol.
 #
 
 HOSTNAME=`hostname`
